### PR TITLE
Updated the web debug toolbar panel

### PR DESCRIPTION
**Pull request by javiereguiluz**
This PR updates the bundle to use the new web debug toolbar and profiler panels when using a modern Symfony application.

### Before (when using Symfony 2.8.1)

before

### After (when using Symfony 2.8.1)

after

### CODE REVIEW

File: Resources/views/Collector/connect.html.twig [hunk 1]
```diff
@@ -1,24 +1,49 @@
...
+            <span class="icon">
+                {{ include('@SensioLabsConnect/Collector/icon.png.twig') }}
+            </span>
+            <strong>Connect</strong>
+        </span>
+    {% endif %}
 {% endblock %}
 
 {% block panel %}
+    <h2>SensioLabs Connect Calls</h2>
```
Comment (fabpot) [inline Resources/views/Collector/connect.html.twig]: extra space before Connect should be removed.

**Comment by fabpot** [comment]
:+1:

File: Resources/views/Collector/connect.html.twig (hunk 1 above)
Comment (javiereguiluz) [inline Resources/views/Collector/connect.html.twig]: Indeed. Fixed. Thanks.

**Comment by fabpot** [comment]
Thank you @javiereguiluz.

**Pull request closed without merging**